### ISSUE TITLE
[Feature Request]: Logit Bias for Hugging Face

### ISSUE THREAD

**Issue by Tyrest** [refuel-ai/autolabel#484]
**Is your feature request related to a problem? Please describe.**
This is a continuation of #352 for Hugging Face. We previously added logit biasing for Open AI and since have found it fairly useful in increasing the control users have over the LLM outputs.

**Describe the solution you'd like**
Implement logit biasing for Hugging Face pipelines using [Sequence Bias Logits Processor](https://huggingface.co/docs/transformers/main/en/internal/generation_utils#transformers.SequenceBiasLogitsProcessor).

**Comment by Tyrest** [comment]
Seems like this is not possible yet due to how Hugging Face pipelines are implemented in `langchain` right [now](https://github.com/langchain-ai/langchain/blob/3eb4112a1f00859e667cea1035d3edf28145e966/libs/langchain/langchain/llms/huggingface_pipeline.py#L161).

We would need to pass in `generate_kwargs` as shown [here](https://github.com/huggingface/transformers/blob/d2295708a64feff485bb40caf0decf7bae5a05a2/src/transformers/pipelines/text2text_generation.py#L136) to use sequence biasing.
